backend/src/main/java/com/google/sps/servlets/AddParticipantServlet.java

```diff
@@ -103,15 +116,30 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
       // Match not found, insert participant entity into datastore
       Entity participantEntity = new Entity(KEY_PARTICIPANT);
       participantEntity.setProperty(PROPERTY_USERNAME, username);
-      participantEntity.setProperty(PROPERTY_STARTTIMEAVAILABLE, startTimeAvailable);
-      participantEntity.setProperty(PROPERTY_ENDTIMEAVAILABLE, endTimeAvailable);
+      participantEntity.setProperty(PROPERTY_STARTTIMEAVAILABLE, startTimeAvailable.toString());
+      participantEntity.setProperty(PROPERTY_ENDTIMEAVAILABLE, endTimeAvailable.toString());
       participantEntity.setProperty(PROPERTY_DURATION, duration);
       participantEntity.setProperty(PROPERTY_TIMESTAMP, timestamp);
       datastore.put(participantEntity);
     }
 
-    // Redirect back to the HTML page
-    response.sendRedirect("/index.html");
+    response.setContentType("text/plain;charset=UTF-8");
+    response.getWriter().println("Received form input details!");
+  }
+
+  private JSONObject retrieveRequestBody(HttpServletRequest request, HttpServletResponse response)
```

Comment:
WDYT about having this method only take the request and then throwing the IOException to the caller to handle with the response?